.all-contributorsrc

```diff
@@ -38,6 +38,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "amitkparekh",
```

Comment:
While this looks fine, it is usually easier to do this after the PR by adding a comment to add a contributor.  That way all contributors has control over the updates entirely.  No big deal, just mentioning for next time 🙂

Reply:
Yeah, we'll have a better process in place going forward. I just wanted to get this one out of the way.